shared/elements/gaia_grid/js/items/mozapp.js

```diff
@@ -171,14 +171,23 @@
     get name() {
       var userLang = document.documentElement.lang;
 
+      var getName = function(obj) {
+        if (obj.short_name && obj.short_name.length > 0) {
+          return obj.short_name;
+        }
+        return obj.name;
+      };
+
       if (navigator.mozL10n && userLang in navigator.mozL10n.qps) {
-        return navigator.mozL10n.qps[userLang].translate(this.descriptor.name);
+        return navigator.mozL10n.qps[userLang].
+          translate(getName(this.descriptor));
       }
 
       var locales = this.descriptor.locales;
-      var localized = locales && locales[userLang] && locales[userLang].name;
+      var localized =
+        locales && locales[userLang] && getName(locales[userLang]);
 
-      return localized || this.descriptor.name;
+      return localized || getName(this.descriptor);
```

Comment:
This is just a nit, but I would generally prefer not having a ```getName``` function if possible. I think something like this is probably fine:

```
var d = this.descriptor;
return localized || d.short_name || d.name;
```